helm/cas-ciip-portal/values.yaml

```diff
@@ -45,6 +45,9 @@ env:
 fider:
   host: # TODO move fider template in this repo
 
+mailhog:
+  enable: false
```

Comment:
```suggestion
  enable: false
  resources:
    limits:
      cpu: 100m
      memory: 128Mi
    requests:
      cpu: 50m
      memory: 128Mi
```

Reply:
we need to specify the resources as the default limit is a bit too high on the CPU, and our quotas are not that high